.github/styles/Microsoft/Avoid.yml

```diff
@@ -6,7 +6,6 @@ level: error
 tokens:
   - abortion
   - and so on
-  - and/or
```

Comment:
@seialkali this is not something we want in our styles. We should not be using and/or in docs.

Reply:
Also, I tried an and/or yesterday and the linter said "no" it was bad.  So it won't pass grammar test anyway.

Reply:
Ok no problem, I'll remove!